about.js

```diff
@@ -0,0 +1,23 @@
+// hamburguer menu
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+const navBarMenu = document.querySelector('.navbar-menu');
+const logo = document.querySelector('.logo');
+const main = document.querySelector('.main');
+const footer = document.querySelector('.footer-about');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  navBarMenu.classList.add('display');
+  logo.classList.add('hidden');
+  main.classList.toggle('active');
+  footer.classList.toggle('active');
+});
+
+document.querySelectorAll('.link-menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  main.classList.remove('active');
+  footer.classList.remove('active');
+}));
```

Comment:
- [ ] Kindly consider when the mobile menu is opened make the menu does not scroll to be fixed on screen, you can look at the following image to get what I mean
>![Recording 2023-03-24 at 00 01 12](https://user-images.githubusercontent.com/53402600/227360588-69fcde71-f60f-459e-8c2f-d0fbaa487387.gif)

Reply:
Thank you @omarsalem7 I will make changes according to your suggestions.